### DISCUSSION TITLE
how does one move installed outfits into cargo?

### DISCUSSION THREAD

**Discussion by OCDroppsy** [endless-sky/endless-sky#9885] [Q&A] [answered]
I've salvaged a gun I'd like to try out, but I'd like to keep my old gun in case the new one doesn't work out.
The new gun's in cargo.  I can sell the old gun at an outfitter and then install the new gun, but then I don't seem to be able to re-buy the old gun into cargo.  Am I missing something?
This is v10.6 on Windows.

**Comment by Hecter94** [comment]
If you uncheck "Show Outfits in Cargo", or use Ctrl-click to deselect all of your ships in the outfitter, your UI will change and show your total cargo capacity, you will then buy outfits directly into cargo.

**Comment by xX-Dillinger-Xx** [comment, marked as answer]
@OCDroppsy 
There is one other thing Hector94 left out. You can also leave your old gun in storage at the outfitters.  Select the outfit that is installed and push `u` on your keyboard. This will store it at the outfitters. 
BTW you can always hit `F1` and a "Help" dialogue will pop up. It has a good description about all the options and controls.

**Comment by OCDroppsy** [comment]
THANKS so much for the fast replies!!

The tip about Storage is what I really needed.

Still, even with this info it's confusing.  I can get the outfits into Storage, and from there I can move them to "Cargo", but into which ship's cargo do they go?  Automatically first into the flagship, unless it's full, and then into another fairly random ship in your fleet?

It would be wonderful to have the "U" and "C" shortcuts listed on the screen somewhere.  I looked in the manual for this info before posting here and didn't see it there either.  Maybe I overlooked it.  I think this may've been mentioned in one of the many popups I got when I got my 2nd ship, but there were quite a lot of them and I didn't know enough to understand some of them yet.

**Reply by xX-Dillinger-Xx** [reply]
I believe the distribution of the cargo is somewhat random. However, their is a exception. Let me give an example.

- Your flag ship has 40 tons of free cargo space 
- escort A has 70 tons of free cargo space
- you try to add a 60 ton outfit to cargo
- the outfit only fits in Escort A, so that's where it goes

Different scenario:
- your flag ship has 40 tons free
- escort A has 70 tons free
- you try to add an 80 ton outfit
- your fleet has 110 tons total, but because no single ship has 80 you would not be able to add the 80 ton outfit.

>It would be wonderful to have the "U" and "C" shortcuts listed on the screen somewhere

I believe that they are working something for that or at least plan to. I'm not 100%
